lib/modules/wdpa/pame_importer.rb

```diff
@@ -29,16 +32,17 @@ def self.import
           ps.language   = row[12]
         end
 
-      if protected_area.nil?
-        missing_pas << wdpa_id
-      else
-        PameEvaluation.where({
+      if protected_area.nil? && (restricted == false) # If PameEvaluation doesn’t have a PA and isn’t restricted it should be deleted.
```

Comment:
Could you please add the relevant comments just before the line they refer to rather then within the same line? 
Also, when it's just about one atomic boolean condition, I feel like parenthesis are not really necessary to encapsulate that.

Both are for readibility reasons.
Same in the `elsif` condition below. Exception made for the first group of paranthesis which are encapsulating boolean conditions correctly.

Reply:
Thanks @Levia I will fix this!